helm/reana/templates/reana-server.yaml

```diff
@@ -40,6 +40,10 @@ spec:
         args: ["set -e; ./scripts/setup; invenio run -h 0.0.0.0 -p 5000"]
         tty: true
         stdin: true
+        {{- else }}
+        command: ["/bin/sh", "-c"]
+        args: ["./scripts/setup > /var/log/reana-server-init-output.log 2>&1 &&
+                uwsgi --wsgi-disable-file-wrapper --ini /var/reana/uwsgi/uwsgi.ini"]
```

Comment:
Can we also add to the configuration file the `--wsgi-disable-file-wrapper` option?

Reply:
Are you sure that is going to work? I couldn't find that option [on the docs](https://uwsgi-docs.readthedocs.io/en/latest/Options.html), that's why I left it in the command.